install.sh

```diff
@@ -413,53 +413,48 @@ _tail() {
 
 ###################################################
 # Alternative to timeout command
-# Globals: None
+# Globals: 1 function
+#   _bash_sleep
 # Arguments: 1 and rest
 #   ${1} = amount of time to sleep
 #   rest = command to execute
 # Result: Read description
 # Reference:
-#   https://stackoverflow.com/a/11056286
+#   https://stackoverflow.com/a/24416732
 ###################################################
 _timeout() {
-    declare -i sleep="${1}" && shift
-    declare -i pid watcher
-    {
-        { "${@}"; } &
-        pid="${!}"
-        { read -r -t "${sleep:-10}" && kill -HUP "${pid}"; } &
-        watcher="${!}"
-        if wait "${pid}" 2> /dev/null; then
-            kill -9 "${watcher}"
-            return 0
-        else
-            return 1
-        fi
-    } &> /dev/null
+    declare timeout="${1:?Error: Specify Timeout}"
+    shift
+    declare cmd="${@}"
```

Comment:
Codacy found an issue: [Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=5837421)